Sources/Plot/API/HTMLComponents.swift

```diff
@@ -579,6 +579,72 @@ extension List: ComponentContainer where Items == ComponentGroup {
     }
 }
 
+// Component used to render a `<picture>` element for responsive image support.
+public struct Picture: Component {
+
+    /// Image to use as fallback.
+    public var image: Image
+    /// A set of `source` elments from which a user agent will chose based on its requirements.
+    public var sources: [Source]
+
+    /// Initialize a `Picture` component with multiple `source` elements.
+    internal init(image: Image, sources: [Picture.Source] = []) {
+        self.image = image
+        self.sources = sources
+    }
+
+    /// Initialize a `Picture` component with a single `source` element.
+    /// Image to use as fallback.
+    /// A set of `source` elments from which a user agent will chose based on its requirements.
+    public init(image: Plot.Image, source: Source) {
```

Comment:
This initializer does feel a bit unnecessary to include in the library, given by above comment. But would be happy to change my mind if you think it's indeed useful, @majortom64.

Reply:
Also `Plot.Image` can become just `Image` here as well, if we do decide to keep this initializer.

Reply:
Yup, will correct the Plot.Image.